datalabsupport/mail/tests.py

```diff
@@ -56,7 +56,7 @@ def test_message_link_formatting(self, mock_slack):
 
         get_messages(folder='INBOX', channel='#mailtest')
         expected = ("<http://info.sagepub.co.uk/c/"
-                    "11o8zNX6zKkZTZfhD08iisMdxM3A|submission "
+                    "11o8zNX6zKkZTZfhD08iisMdxM3A|_blank|submission "
```

Comment:
@sebbacon this change makes the test pass, not sure if I need to dig in any further?

Reply:
the HTML goes into `lxml` via `BeautifulSoup`, seems like it's just a minor formatting issue & can ignore?

Reply:
I remember this being very hairy, but I haven't time to think about it again! :shipit:  and let's worry another time!

Reply:
👍